src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -581,11 +586,13 @@ public Boolean apply(WebDriver driver) {
                 .executeScript("return !!(" + script + ");");
           } catch (WebDriverException e) {
             PageObjectLogging.logError("waitForJavaScriptTruthy", e);
+            PageObjectLogging.log("Checking script to be truthy", script, false);
             return false;
           }
         }
       });
     } finally {
+      PageObjectLogging.log("Checking script to be truthy", script, true);
```

Comment:
`catch` and `finally` blocks will be executed if we throw `WebDriverException`, so, we will have 2 log messages:
```
PageObjectLogging.log("Checking script to be truthy", script, false);
PageObjectLogging.log("Checking script to be truthy", script, true);
```
Is it desired behaviour?